test/e2e-common.sh

```diff
@@ -146,6 +146,21 @@ function build_components_from_source() {
   return $?
 }
 
+function build_control_plane_from_source() {
+
+  # Remove existing control plane artifacts if they exist
+  [ -f "${EVENTING_KAFKA_CONTROL_PLANE_ARTIFACT}" ] && rm "${EVENTING_KAFKA_CONTROL_PLANE_ARTIFACT}"
+  [ -f "${EVENTING_KAFKA_POST_INSTALL_ARTIFACT}" ] && rm "${EVENTING_KAFKA_POST_INSTALL_ARTIFACT}"
+
+  header "Control plane setup"
+  control_plane_setup || fail_test "Failed to set up control plane components"
+
+  header "Building Monitoring artifacts"
+  build_monitoring_artifacts || fail_test "Failed to create monitoring artifacts"
+
+  return $?
```

Comment:
After this, you will need to install the newly created artifacts (as you can see in https://github.com/knative-extensions/eventing-kafka-broker/blob/main/test/e2e-common.sh#L182-L196

Specifically, we need the control plane artifact, tls network artifact, and the control plane artifact. We should also restore the test config in the same was as is being done in the linked code